Utilities/bootstrap

```diff
@@ -520,6 +521,7 @@ def build_with_cmake(args, cmake_args, ninja_args, source_path, build_dir, cmake
 
     if platform.system() == 'Darwin':
         call(["sed", "-i", "", "s/macosx10.10/macosx%s/" % (g_macos_deployment_target), "build.ninja"], cwd=build_dir)
+        call(["sed", "-i", "", "s/CompilerPluginSupport.swiftinterface .*pm\/ManifestAPI\/PackageDescription.swiftinterface/CompilerPluginSupport.swiftinterface/", "build.ninja"], cwd=build_dir)
```

Comment:
Looks like the way we're building `PackageDescription` with CMake is fairly broken, we are forcing it to produce a Swift interface, but CMake actually doesn't know about them. We haven't noticed the breakage until now, but since `CompilerPluginSupport` needs to link it, it surfaces here and needs to be worked around.